guides/release/contributing/index.md

```diff
@@ -23,6 +23,18 @@ Have a look at the [list of main repositories](repositories) to learn about the
 
 If you need some inspiration, you can check out the [Help Wanted](https://help-wanted.emberjs.com/) dashboard to browse for issues. If you are a beginner, look out for issues with the "Help wanted" and "Good first issue" labels.
 
+# Changing the API documentation
+
+To change the [API reference guidance](https://api.emberjs.com/ember/release), find the relevant section for the *package*, *method* or *class* you wish to change.
+At the top of the section, you will find a link to the 'defined in' or 'inherited from' file.
+Following this link will open the file where the guidance and the code for the function is defined.
```

Comment:
Sample screenshot:

<img width="1200" alt="" src="https://user-images.githubusercontent.com/16869656/122662611-9def4200-d194-11eb-838b-459e87090aec.png">

A possible update to the paragraph:

```suggestion
In the [API Guides](https://api.emberjs.com/ember/release), suppose you found a typo or wish to provide more details for a package, method, or class. Here is how you can make a change.

At the top of the page (for the package, method, or class), you will find the words "Defined in" or "Inherited from". Next to these words is a link to the source code on GitHub.

[Insert screenshot and provide alternate text if needed]

You can open the link to find a comment block. Make a pull request to update the comment block. The API Guides will be updated in the near future (e.g. on the next Ember release).
```

Reply:
@fozy81 I think adding screenshot(s) will help us concisely and precisely describe what a person will want to do to update the API Guides.

I made quite some changes to your text. Could you have a look at the suggested text, to see if it's okay with you? (Feel free to suggest something else.)

@jenweber Is it correct that, _in general_:

- Once a pull request to update the comment block is merged, the API Guides will be updated not immediately, but at the next Ember release.

Reply:
I don’t think the change would necessarily be the next release since things bake in beta for 6 weeks. I’d use words like “future release” instead of “next release” to be more precise.

@ijlee2  what do you think about screenshots being optional for this to be merged? I kinda like keeping this section short and sweet for now.

Reply:
@jenweber Yep, I think we can make adding screenshots optional. 🙂